src/Tree/index.tsx

```diff
@@ -138,7 +139,7 @@ class Tree extends React.Component<TreeProps, TreeState> {
    * specified in `props.scaleExtent`.
    */
   bindZoomListener(props: TreeProps) {
-    const { zoomable, scaleExtent, translate, zoom, onUpdate } = props;
+    const { zoomable, scaleExtent, translate, zoom, onUpdate, interactiveNode } = props;
```

Comment:
See comment on `src/Tree/types.ts`

```suggestion
    const { zoomable, scaleExtent, translate, zoom, onUpdate, hasInteractiveNodes } = props;
```